spec/rubyfit/message_writer_spec.rb

```diff
@@ -128,13 +128,16 @@ def check_bytes(bytes, expected_fields, leftover = 0)
         0, # Padding
         1, # Big endian
         0, 20, # Global message number
-        5, # Field count
+        8, # Field count
         # Fields are 3 bytes each - field ID, size in bytes, type ID
         253, 4, 134, # timestamp
         0, 4, 133, # position lat
         1, 4, 133, # position long
         5, 4, 134, # distance
         2, 2, 132, # elevation
+        3, 1, 2, # heart_rate
+        4, 1, 2, # cadence
+        7, 2, 132, # power
```

Comment:
I believe this is added to clean up my old activity types PR which did not update the test: https://github.com/ridewithgps/rubyfit/pull/17

Thanks!